src/main/native/ECKey.c

```diff
@@ -2121,12 +2121,14 @@ JNIEXPORT jbyteArray JNICALL Java_com_ibm_crypto_plus_provider_ock_NativeInterfa
   jbyteArray           retSecretBytes = NULL;
   size_t               secret_key_len = 0;
 
-  
-  if( debug ) gslogFunctionEntry(functionName);
+  if( debug ) {
```

Comment:
Can we fix the spaces in the parentheses too?

Reply:
Removed the spaces in the parentheses here and other locations in this method.

In the long run we should prob decide on some sort of tooling or set of rules that can be used to reformat all of the C code to make it more consistant. We currently have the style.xml settings which are available and documented in the README file however this set of rules does not apply to C code. Enforing style via tooling is much easier and effecient then through code reviews :)